st2common/st2common/content/bootstrap.py

```diff
@@ -19,21 +19,12 @@
 
 from oslo_config import cfg
 
-import st2common
 from st2common import config
 from st2common.script_setup import setup as common_setup
 from st2common.script_setup import teardown as common_teardown
 from st2common.bootstrap.base import ResourceRegistrar
-import st2common.bootstrap.triggersregistrar as triggers_registrar
```

Comment:
We're using the same registrars in #2842, but instead of calling them from bootstrap script, we're importing them in api. At some point I would like us to rewrite the whole thing with networkx to properly assign the dependencies between different types of content.